### PR TITLE
ddl, statistics: update the stats table after creating/ droping column.

### PR DESCRIPTION
**Pull request by hanfei1991**
After ceating a col, we can insert a record to stats_histogram and insert a bucket record to stats_bucket with default value.
After droping a col, we just remove all records about this column.
Both operations will update version. After #3059 , it will not cost too much.
@shenli @coocood @zimulala @lamxTyler PTAL

### CODE REVIEW

File: statistics/ddl.go [hunk 1]
```diff
@@ -48,17 +56,85 @@ func (h *Handle) insertTableStats2KV(info *model.TableInfo) error {
...
+		rs, err := h.ctx.(sqlexec.SQLExecutor).Execute(fmt.Sprintf("select count from mysql.stats_meta where table_id = %d", tableID))
+		if err != nil {
+			return errors.Trace(err)
+		}
+		row, err := rs[0].Next()
+		if err != nil {
+			return errors.Trace(err)
+		}
+		count := row.Data[0].GetInt64()
+		valueBytes, err := codec.EncodeValue(nil, types.NewDatum(colInfo.DefaultValue))
```
Comment (coocood) [inline statistics/ddl.go]: The default value may be current timestamp.
Use `OriginDefaultValue`.

**Comment by hanfei1991** [comment]
@coocood Updated, PTAL

**Comment by lamxTyler** [comment]
LGTM

File: ddl/column.go [hunk 2]
```diff
@@ -169,6 +169,14 @@ func (d *ddl) onAddColumn(t *meta.Meta, job *model.Job) error {
...
 		job.State = model.JobDone
 		job.BinlogInfo.AddTableInfo(ver, tblInfo)
+
+		// Send the event to stats handle.
+		if d.ddlEventCh != nil {
+			select {
+			case d.ddlEventCh <- &Event{Tp: model.ActionAddColumn, TableInfo: tblInfo, ColumnInfo: columnInfo}:
+			default:
+			}
+		}
```
Comment (zimulala) [inline ddl/column.go]: Extract this code into a function？ Many places use it.

Comment (hanfei1991) [inline ddl/column.go]: Good suggestion.

**Comment by hanfei1991** [comment]
@zimulala @coocood PTAL

**Pull request merged**